grammar.js

```diff
@@ -63,6 +62,10 @@ module.exports = grammar({
     $.literal,
   ],
 
+  precedences: $ => [
```

Comment:
Interesting. I wasn't aware of this feature. Could you add a comment linking to a doc (maybe it's just https://github.com/tree-sitter/tree-sitter/pull/939 for now?) for this?

Reply:
Sure!
These relative precedences seem to bring more transparency compared to global numeric precedences